docs/moderne-cli/README.md

```diff
@@ -117,31 +198,33 @@ Now that the repository is configured, it's time to migrate it to Spring Boot 3
        ▛▀▀█▀▛▀▀▀▀▜
        ▌▟▀  ▛▀▀▀▀▜
        ▀▀▀▀▀▀▀▀▀▀▀
-   Moderne CLI v0.3.2
+   Moderne CLI VERSION_CLI
 
    > Selecting repositories
 
-   > spring-projects/spring-petclinic@2.0.0
-   Selected 1 repositories (0.02s)
-   
+   > spring-projects/spring-petclinic@main
```

Comment:
Hmm this looks like a different branch & project; I had thought we'd use https://github.com/moderneinc/spring-petclinic/tree/2.0.0 instead to have an intentionally outdated project & branch.

Reply:
This is the project we provide in the instructions above, though. We also explicitly call out to use the b527de52f5fd19f9fe550372c017d145a3b2a809 commit. If you want it to be different that's fine -- but that's how I've always run this.